Contributions.md

```diff
@@ -0,0 +1,51 @@
+# Jore-map-ui conventions
+
+## Git conventions
+
+* When making a PR, write closes #{*issue_id(s)*} or resolves #{*issue_id(s)*}
+* Branch name: {*issue_number*}-{*describing_name*}
+
+**Git columns**
+
+* Backlog *- tasks that are blocked or need planning*
+* To do *- tasks that aren't blocked. These tasks should be in priority order.*
+* Sprint *- tasks to do in the current sprint*
+* In progress *- tasks that are in progress or need review fixes*
+* Done sprint *- completed tasks in the current sprint*
+* Done *- all completed tasks*
```

Comment:
Shouldn't this be "completed tasks from previous sprints" so you get the sense of moving from "Done sprint" to "Done". "all completed tasks" implies that the tasks done in current sprint are also included in "Done".